internal/controllers/utils/constants.go

```diff
@@ -68,6 +72,20 @@ const (
 	defaultServiceCAFile    = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt" // nolint: gosec // hardcoded path only
 )
 
+// Default timeout values
+const (
+	DefaultHardwareProvisioningTimeout = 90 * time.Minute
+	DefaultClusterProvisioningTimeout  = 90 * time.Minute
+	DefaultClusterConfigurationTimeout = 30 * time.Minute
+)
+
+// Timeout config keys
```

Comment:
It would be helpful to add more information to the comment: that each of these areoptional keys in the respective ConfigMaps such that timeout is configured for each operation.